README.md

```diff
@@ -157,6 +157,7 @@ Option | Type | Default | Description
 |[`inline`](#inline) | `Boolean` | `true` | Render input field and selected tags in-line
 |[`allowUnique`](#allowUnique) | `Boolean` | `true` | Boolean value to control whether tags should be unqiue
 |[`allowDragDrop`](#allowDragDrop) | `Boolean` | `true` | Boolean value to control whether tags should have drag-n-drop features enabled
+|[`renderSuggestion`](#renderSuggestion) | `Function` | `undefined` | Function that returns a custom suggestion for overriding the default one
```

Comment:
`Render prop for rendering your own suggestions`

Reply:
done